.README/rules/check-lines-alignment.md

```diff
@@ -0,0 +1,52 @@
+### `check-lines-alignment`
+
+Reports invalid alignment of JSDoc block lines. This is a
+[standard recommended to WordPress code](https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/javascript/#aligning-comments), for example.
+
```

Comment:
As per #483 , it seems the community is more in favor of a rule to prevent this kind of spacing, so while I support having the functionality you are providing as an option, and giving the rationale related to WordPress as you have, I'd like to change the configuration a little.

In case you don't feel you have the time to add the opposite functionality (i.e., preventing extra spaces for alignment), I think we should at least design the rule to accept a string option, "never" or "always", where "never" is the default, but where we throw an exception for the time being if the option is "never" indicating that the default is not yet implemented. This can avoid a future breaking change for those who wish to use the rule now.

Until such a "never" option may be implemented, of course, we wouldn't want this rule added to our `recommended` config, but we can add it as an opt-in rule, with docs indicating that only the non-default "always" option is currently implemented.

Reply:
Makes sense! I implemented the option here: https://github.com/gajus/eslint-plugin-jsdoc/pull/636/commits/07e93f7ddb9a9b52d1df92e88dc6d5ca42987b39

I pushed with `--no-verify` for now, so you can see the changes. But I still need to test the throw to achieve the full coverage. I'll think of an approach to test it and I'll amend the commit.